src/app/pages/Designathons/Designathon24/Designathon24.js

```diff
@@ -0,0 +1,144 @@
+import { Icon, Section } from "app/Symbols";
+import { Text } from "../../../components";
+import WINNERS_2024 from "assets/data/designathon/2024/winners.json";
+import TIMER_2024 from "assets/data/designathon/2024/timer.json";
+import JUDGES_2024 from "assets/data/designathon/2024/judges.json";
+import KEYNOTE_2024 from "assets/data/designathon/2024/keynote.json";
+import PRIZES_2024 from "assets/data/designathon/2024/prizes.json";
+import WORKSHOP_HOSTS_2024 from "assets/data/designathon/2024/workshop-hosts.json";
+import RULES_2024 from "assets/data/designathon/2024/rules.json";
+import cn from "./Designathon24.module.scss";
+import FOF from "./assets/FOF.png";
+import notion from "./assets/notion.png";
+import balsamiq from "./assets/balsamiq.png";
+import {
+	Profile,
+	Prizes,
+	SectionNavigation,
+	FAQ,
+	Rules,
+	Timer,
+	WinnerShowcase,
+} from "./components";
+import Schedule from "./components/Schedule";
+import { Splash } from "./components/Splash";
+
+const Designathon24 = () => (
+	<main className={cn.container}>
+		<div className={cn.hero}>
+			<div style={{ height: "100%", width: "100%" }}>
+				<Splash />
+			</div>
+			<div className={cn.title}>
+				<Icon className={cn.icon} src="designathon-white.svg" w="96" h="96" />
+				<Text size="XXXL" className="bold">
+					True to You
+				</Text>
+				<div className={cn.timer}>
+					<Timer breakpoints={TIMER_2024} />
+				</div>
+			</div>
+		</div>
+
+		<SectionNavigation />
+		<WinnerShowcase winners={WINNERS_2024} />
+
+		<div className={cn.about} id="s-about">
+			<Section>
+				<h1 className={cn.aboutHeading}>About</h1>
```

Comment:
Chore [repeating]: I know this was copied from before, but heading level 1 should be reserved for the main page title whereas subsequent section headings should be hierarchy below.